app/views/expenses_manager/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= stylesheet_link_tag "application" %>
+<%= stylesheet_link_tag "expense" %>
+
+<%= render "layouts/nav_bar", {prev_path: nil, title: 'new expense', from_form: false, next_path: categories_path, next_text: 'search' } %>
+
+<div class="expense-form">
+    <%= render 'form', {category: @category, expense: @expense, submit_path: category_expenses_manager_index_path(@category, @expense)} %>
+</div>
```

Comment:
- [ ] Please make sure when the user clicks on the "Back" button (<), the user is directed to the transactions page for that category as requested in the project requirements. Currently, the user is redirected to the home page(categories page).

![image](https://github.com/patriciachrysy/expenses-control/assets/83803414/c15febcf-74a5-41a9-acfb-64745f22d981)